app/helpers/pre_built_pages_helper.rb

```diff
@@ -1,14 +1,15 @@
 module PreBuiltPagesHelper
-  def include_page_content(path, default = nil)
+  def include_page_content(path, default = nil, postfix = "")
     page = Page.find_by(path: path)
     return default if page.blank?
-    page.render_content
+    page.render_content + postfix
   end
 
-  def include_or_create_page_content(path, message)
+  def include_or_create_page_content(path, message, options = {})
     include_page_content(
       path,
-      "<div><a href='#{edit_page_path(path)}'>#{message}</a></div>"
+      "<div><a href='#{edit_page_path(path)}'>#{message}</a></div>",
+      options[:edit_link] ? "<div class=edit_link><a href='#{edit_page_path(path)}'>このページを編集</a></div>" : ""
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.